### PR TITLE
Unit testing code for modules/sonic_config.py added

### PR DESCRIPTION
**Pull request by kannankvs**
##### SUMMARY
Unit testing code for modules/sonic_config.py added

##### ISSUE TYPE
- Feature Pull Request

##### COMPONENT NAME
sonic_config Unit Testing

### CODE REVIEW

File: tests/unit/modules/network/sonic/fixtures/sonic_config.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+---
+merged_01:
+  module_args:
+    commands: ['snmp-server community public group ro']
```
Comment (kerry-meyer) [inline tests/unit/modules/network/sonic/fixtures/sonic_config.yaml:4]: There is no "commands" argument defined for the sonic_config module. What is the purpose of this line and test?

Comment (kerry-meyer) [reply tests/unit/modules/network/sonic/fixtures/sonic_config.yaml:4]: Okay. In looking at the module definition, I see that "commands" is an alias for the "lines" argument.

I will resolve this issue.

**Review by kerry-meyer — CHANGES_REQUESTED** [review]
These changes provide useful testing and they look good. I have posted a request to clean up the saved configuration as an additional section of the test.

File: tests/unit/modules/network/sonic/fixtures/sonic_config.yaml [hunk 2]
```diff
@@ -3,6 +3,9 @@ merged_01:
   module_args:
     commands: ['snmp-server community public group ro']
     save: yes
+  expected_commands_to_device:
+    - snmp-server community public group ro
+    - write memory
```
[the remark this answers is not in the document]
Comment (kannankvs) [reply tests/unit/modules/network/sonic/fixtures/sonic_config.yaml:8]: @kerry-meyer : There is no need of that. Whatever is passed in "commands" is passed asis to the device. Even if we give "no" form of command, that also goes to device asis. Even if we pass any junk values inside commands, it  goes to device asis.

[the remark this answers is not in the document]
Comment (kerry-meyer) [reply tests/unit/modules/network/sonic/fixtures/sonic_config.yaml:8]: I realize now that the receiving side is mocked and nothing actually goes to the device. I will mark this issue "Resolved" and Approve and merge the PR.

**Review by kerry-meyer — APPROVED** [review]

**Pull request merged**